caboose.go

```diff
@@ -122,7 +130,16 @@ type ErrCoolDown struct {
 }
 
 func (e *ErrCoolDown) Error() string {
-	return fmt.Sprintf("multiple saturn retrieval failures seen for CID %s/Path %s, please retry after %s", e.Cid, e.Path, humanRetry(e.retryAfter))
```

Comment:
:information_source:  This was producing very confusing log errors `failures seen for CID b/Path` due to `e.Cid` being undefined.